facts/test_datasources.py

```diff
@@ -108,6 +108,28 @@ def test_switchtypes_tsv():
     assert result, err
 
 
+def test_switchconfigs_tsv():
+    """test switchconfigs"""
+
+    directory = "../switch-configuration/config/types/"
+    for filename in os.listdir(directory):
+        meta = {
+            "file": directory + filename,
+            "header": False,
+            "count": "1+",
```

Comment:
I think the count here should be `4+` since it looks like we have a mix of `4-6` columns for the config types